django-backend/fecfiler/transactions/managers.py

```diff
@@ -64,9 +64,18 @@ def get_queryset(self):
             date=OuterRef("date"), created__lte=OuterRef("created")
         )
         group_clause = Q(aggregation_group=OuterRef("aggregation_group"))
+        force_unaggregated_clause = Q(force_unaggregated=False) | Q(
```

Comment:
could be ~Q(force_unaggregated=True)

Reply:
When working on the loan/debt pull forward, I found that ~Q(some_field=True) did not include the records with NULL, just the records with False. Do you not find this to not be the case? I will double-check.

Reply:
Made this correction and updated pull forward query as well.